scripts/check-recipe-code.sh

```
@@ -0,0 +1,158 @@
+if [ ! -d "$RECIPES" ]; then
+  echo "No recipes directory at $RECIPES"
+  exit 1
+fi
+
+echo "==> Building and packing pushduck"
+( cd "$REPO_ROOT/packages/pushduck" && pnpm build >/dev/null 2>&1 ) || {
+  echo "package build failed"; exit 1;
+}
+rm -rf "$WORK" && mkdir -p "$WORK/blocks" "$WORK/app-stubs/lib"
+( cd "$REPO_ROOT/packages/pushduck" && npm pack --pack-destination "$WORK" >/dev/null 2>&1 )
+TARBALL="$(ls "$WORK"/pushduck-*.tgz | head -1)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)